### DISCUSSION TITLE
这是属于前端的技术嘛？

### DISCUSSION THREAD

**Discussion by ChinaXiaoMo** [share121/inter-knot#1570] [其他]
这是属于前端的技术嘛？
这是属于前端的技术嘛？
这是属于前端的技术嘛？
这是属于前端的技术嘛？
这是属于前端的技术嘛？
这是属于前端的技术嘛？
这是属于前端的技术嘛？
这是属于前端的技术嘛？
这是属于前端的技术嘛？
想成为一名前端工程师！！
![server-icon](https://github.com/user-attachments/assets/cbd9c62f-5393-44b6-9e87-3ac61e41f6f2)

**Comment by Myitian** [comment]
学习前端，可以用MDN
[Web 开发技术 | MDN
![图片](https://github.com/user-attachments/assets/fec49525-05c3-4227-9809-f395c3214475)
](https://developer.mozilla.org/zh-CN/docs/Web)

**Comment by Mayf1ower** [comment]
是的这属于前端技术
想学习前端可以从HTML CSS JavaScript开始学起，这是最基本的
